contracts/StablecoinConverter.sol

```diff
@@ -175,18 +186,35 @@ contract StablecoinConverter is EpochTokenLocker {
                 volumes[i]
             );
         }
+        int fee = tokenConservation[0];
+        require(fee < auctionData[batchIndex].feeCollected, "Fee is not higher than before");
```

Comment:
I am not sure this error message is complete. "Before" what?

Maybe something like "Collected fee must exceed fee requested"?